node/packages/parser/src/ethereum/__tests__/mockData/ethOnly.ts

```diff
@@ -1,82 +1,53 @@
+      isAddress: true,
+    },
+  ],
+  vout: [
+    {
+      value: '100295077000000000',
+      n: 0,
+      addresses: ['0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D'],
+      isAddress: true,
     },
-    ethereumSpecific: {
-      status: -1,
-      nonce: 6,
-      gasLimit: 140096,
-      gasUsed: null,
-      gasPrice: '24200000000',
-      data:
+  ],
+  blockHeight: -1,
+  confirmations: 0,
+  blockTime: 1624048855,
+  value: '100295077000000000',
+  fees: '0',
+  rbf: true,
+  coinSpecificData: {
+    tx: {
+      nonce: '0x6',
+      gasPrice: '0x5a26eb200',
+      gas: '0x22340',
+      to: '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D',
+      value: '0x16451d7554e3200',
+      input:
         '0x7ff36ab500000000000000000000000000000000000000000016df982add888d233e7e8c000000000000000000000000000000000000000000000000000000000000008000000000000000000000000070336a5a868cf9e0a9d1b47fbe1f521ad6c5c3640000000000000000000000000000000000000000000000000000000060cd0b8f0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc200000000000000000000000095ad61b0a150d79219dcf64e1e6cc01f0b64c4ce',
+      hash: '0xc3acd7001e1d01fe4b451e33e24ce399ee533a18d00b441e3bb6290209b072e1',
+      blockNumber: '',
+      from: '0x70336A5A868cF9E0a9D1B47fbE1F521aD6C5C364',
+      transactionIndex: '',
     },
   },
-  tx: {
-    txid: '0x5557426ee8e2f3da4703a4b5ab265e5d3a5626da1f69b4d048c7230a75fa5936',
-    vin: [
-      {
-        n: 0,
-        addresses: ['0x8C8D7C46219D9205f056f28fee5950aD564d7465'],
-        isAddress: true,
-      },
-    ],
-    vout: [
-      {
-        value: '46224000000000000',
-        n: 0,
-        addresses: ['0x05EFB6161fe280dC17E5f9EaB6200b889E2D0A88'],
-        isAddress: true,
-      },
-    ],
-    blockHash: '0x39fbd8a96a5b519ea5301ecde6dbf2c63c4a767a3368586ca4c5165ef0999af3',
-    blockHeight: 12659161,
-    confirmations: 14,
-    blockTime: 1624029849,
-    value: '46224000000000000',
-    fees: '1050000000000000',
-    ethereumSpecific: {
-      status: 1,
-      nonce: 4605,
-      gasLimit: 21000,
-      gasUsed: 21000,
-      gasPrice: '50000000000',
-      data: '0x',
-    },
+  ethereumSpecific: {
+    status: -1,
+    nonce: 6,
+    gasLimit: 140096,
+    gasUsed: null,
+    gasPrice: '24200000000',
+    data:
+      '0x7ff36ab500000000000000000000000000000000000000000016df982add888d233e7e8c000000000000000000000000000000000000000000000000000000000000008000000000000000000000000070336a5a868cf9e0a9d1b47fbe1f521ad6c5c3640000000000000000000000000000000000000000000000000000000060cd0b8f0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc200000000000000000000000095ad61b0a150d79219dcf64e1e6cc01f0b64c4ce',
   },
 }
+
+export default {
+  tx: ethOnly,
+  txMempool: { ...ethOnly, blockHeight: -1, confirmations: 0, fee: 0, ethereumSpecific: undefined },
```

Comment:
It might be worth just making a mempoolData mock object that we can spread against all of the various mocks to ensure they are written over the same and we don't accidentally forget a field.